modules/administration-guide/pages/extensions-for-microsoft-visual-studio-code-open-source.adoc

```diff
@@ -7,12 +7,14 @@
 [id="extensions-for-microsoft-visual-studio-code-open-source"]
 = Extensions for Microsoft Visual Studio Code - Open Source
 
-A paragraph or more of conceptual information about the embedded link:https://open-vsx.org/[Open VSX Registry] that Che admins need to understand.
+To manage extensions for link:https://github.com/microsoft/vscode[Microsoft Visual Studio Code - Open Source IDE], {prod-short} uses one of the link:https://open-vsx.org/about[Open VSX] registry instances:
 
-Explain the process and UX implications for Che end-users.
+* Public link:https://open-vsx.org[open-vsx.org].
+* The embedded, locally cached instance of the Open VSX registry that runs in the `plugin-registry` pod of {prod-short} to support air-gapped, offline, and proxy-restricted environments, containing only a subset of the extensions published on link:https://open-vsx.org[open-vsx.org].
+* Your own custom build of the embedded Open VSX registry for use in workspaces of your organization's {prod-short}.
 
-include::partial$proc_importing-extensions-from-the-open-vsx-registry.adoc[leveloffset=+1]
+The Open VSX registry instance default differs depending on how proxies or DNS are configued: If {prod-short} resolves `pass:c,a,q[https://open-vsx.org]`, then the default is link:https://open-vsx.org[open-vsx.org]. If not, then the default is the embedded, locally cached Open VSX registry instance within the `plugin-registry` pod of {prod-short}.
```

Comment:
```suggestion
If it can be resolved from within your cluster, the default Open VSX registry at `pass:c,a,q[https://open-vsx.org]` will be used; if not, default is the embedded, locally cached Open VSX registry within the {prod-short} `plugin-registry` pod.
```

Reply:
@nickboldt, thank you, Nick. I'm committing your suggestion.